.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: java
-jdk:
-  openjdk8
+jdk: openjdk8
+os: linux
```

Comment:
what does setting this to linux do?

Reply:
When we deploy it to heroku, it spins up a vm (like what was discussed in our latest lecture in this class) with linux as the os and it'll run our app in this container. Linux is the default if we don't specify anything, but it gives a warning in heroku so this gets rid of the warning.

More info if needed: Since it's a whole new image that they spin up, they also have to download all the stuff required to run the app like java, our dependencies, etc. This is reflected in the travis-ci logs when it builds. More specifically, they deploy it to an amazon EC2 which is the same thing as Docker but EC2 is a service and Docker is something we can download and use ourselves on our machine.